decrypt/decrypt_test.go

```diff
@@ -156,3 +156,81 @@ func (suite *DecryptTests) TestDecrypt() {
 	assert.NoError(suite.T(), err, "decrypt failed unexpectedly")
 	os.Args = nil
 }
+
+func (suite *DecryptTests) TestDecryptMultipleFiles() {
```

Comment:
Either expand this test or create another one to cover the following case:
* ForceOverwright = true
* overwrite an empty file and verify that the file has new contents

Reply:
Added in the commit 42c404e

The failure of the s3 integration test is caused by the PR https://github.com/neicnordic/sensitive-data-archive/pull/1046. It seems the image is not successfully created for the tag v0.4.0